CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# CHANGELOG
+Change Log
+==========
 
-TODO: add your changelog
+Version 0.1.48 *(2017-05-19)*
+----------------------------
```

Comment:
I think this line shouldn't be there. because it's the "changelog" of the version